src/helpers/helpers.options.js

```diff
@@ -65,6 +66,46 @@ module.exports = {
 		};
 	},
 
+	/**
+	 * Converts the given font object into a CSS font string.
+	 * @param {Object} font - A font object.
+	 * @return {Stringg} The CSS font string. See https://developer.mozilla.org/en-US/docs/Web/CSS/font
+	 */
+	toFontString: function(font) {
```

Comment:
I would not expose this method to the public API until we need to, but instead I would declare it outside the export (we can test this method via `_parseFont`). We should also deprecate [`helpers.fontString`](/chartjs/Chart.js/blob/master/src/core/core.helpers.js#L592), which could be moved at the end of this file and fallback to this method (similar to what's done in the [canvas helpers](/chartjs/Chart.js/blob/master/src/helpers/helpers.canvas.js#L214))

Reply:
`helpers.fontString` is call from core.tooltip. I think `tooltips.{body|title|footer}Font{Family|Size|Style|Color}` should be replaced with `tooltips.*.font.*` in v3, but `_parseFont` cannot be used for these options in the current version. Should we leave `helpers.fontString` for now?

Reply:
That's fine to keep using `fontString` in this case, though I would still move `toFontString` outside the export to make it fully private for now. What do you think?

Reply:
Yes, I agree.